<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImportScreenshotDialog</class>
 <widget class="QWidget" name="ImportScreenshotDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>675</width>
    <height>688</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>YaRXP import Screenshot</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Path:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="combo_tileset"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Tileset:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="line_path"/>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="button_browse_img">
       <property name="text">
        <string>Browse</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="1">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Basic Positioning and Scaling</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0" colspan="2">
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="3" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Top:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QSpinBox" name="spin_top">
          <property name="minimum">
           <number>-128</number>
          </property>
          <property name="maximum">
           <number>128</number>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Left:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="3">
         <widget class="QSpinBox" name="spin_right">
          <property name="minimum">
           <number>-128</number>
          </property>
          <property name="maximum">
           <number>128</number>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Bottom:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QSpinBox" name="spin_left">
          <property name="minimum">
           <number>-128</number>
          </property>
          <property name="maximum">
           <number>128</number>
          </property>
         </widget>
        </item>
        <item row="3" column="5">
         <widget class="QLabel" name="label_resolution_tiles">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="2" column="5">
         <widget class="QLabel" name="label_resolution_px">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="2" column="4">
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>Resolution (px):</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Right:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="3">
         <widget class="QSpinBox" name="spin_bottom">
          <property name="minimum">
           <number>-128</number>
          </property>
          <property name="maximum">
           <number>128</number>
          </property>
         </widget>
        </item>
        <item row="3" column="4">
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>Resolution (Tiles):</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="6">
         <widget class="QCheckBox" name="check_gridlines">
          <property name="text">
           <string>Show Grid Lines</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="6">
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="2" column="0" colspan="2">
       <layout class="QGridLayout" name="gridLayout_8">
        <item row="0" column="0">
         <widget class="QRadioButton" name="radio_scale_width">
          <property name="text">
           <string>Scale Width (Tiles):</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QSpinBox" name="spin_scale_x">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>1000</number>
          </property>
          <property name="value">
           <number>100</number>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="0">
         <widget class="QRadioButton" name="radio_scale_height">
          <property name="text">
           <string>Scale Height (Tiles):</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSpinBox" name="spin_scale_y">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>1000</number>
          </property>
          <property name="value">
           <number>100</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Processing</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_7">
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="0" column="2">
         <widget class="QPushButton" name="button_similartiles">
          <property name="text">
           <string>Find Similar Tiles</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="1">
         <widget class="QSlider" name="slider_threshold">
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="value">
           <number>80</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Threshold:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QCheckBox" name="check_show_numbers">
          <property name="text">
           <string>Show Tile Numbers</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QPushButton" name="button_show_similar_tiles">
          <property name="text">
           <string>Show Similar Tiles</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QGraphicsView" name="view_orig"/>
   </item>
   <item row="3" column="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="button_import">
       <property name="text">
        <string>Import</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>line_path</tabstop>
  <tabstop>button_browse_img</tabstop>
  <tabstop>combo_tileset</tabstop>
  <tabstop>check_gridlines</tabstop>
  <tabstop>spin_left</tabstop>
  <tabstop>spin_right</tabstop>
  <tabstop>spin_top</tabstop>
  <tabstop>spin_bottom</tabstop>
  <tabstop>slider_threshold</tabstop>
  <tabstop>button_similartiles</tabstop>
  <tabstop>check_show_numbers</tabstop>
  <tabstop>button_show_similar_tiles</tabstop>
  <tabstop>view_orig</tabstop>
  <tabstop>button_import</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>radio_scale_width</sender>
   <signal>toggled(bool)</signal>
   <receiver>spin_scale_x</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>69</x>
     <y>236</y>
    </hint>
    <hint type="destinationlabel">
     <x>178</x>
     <y>235</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radio_scale_height</sender>
   <signal>toggled(bool)</signal>
   <receiver>spin_scale_y</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>139</x>
     <y>279</y>
    </hint>
    <hint type="destinationlabel">
     <x>216</x>
     <y>279</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
